examples/DateRangePickerWrapper.jsx

```diff
@@ -77,6 +80,10 @@ const defaultProps = {
   displayFormat: () => moment.localeData().longDateFormat('L'),
   monthFormat: 'MMMM YYYY',
   phrases: DateRangePickerPhrases,
+
```

Comment:
there shouldn't be two empty lines next to each other; let's collapse this to one